api/part1/standard/sections/clause_16_requirements_class_create_replace_delete.adoc

```diff
@@ -42,7 +42,7 @@ part:: By default, the server SHALL reject a DELETE request on a `System` resour
 
 part:: If the `System` resource is not associated with any `Deployment` resource, and the request contains the `cascade` parameter, the server SHALL accept the DELETE request and delete the `System` resource as well as all its nested resources.
 
-part:: If the `System` resource is associated with a `Deployment` resource, the `Deployment` resource SHALL be deleted first.
+part:: If the `System` resource is associated with a `Deployment` resource, the `Deployment` resource SHALL be be updated, removing the link to the System. If after updating there are no other linked systems in the `Deployment` resource, the `Deployment` SHALL be deleted.
 ====
```

Comment:
Thinking about ti, I don´t think we should delete the Deployment automatically, even if there are no more associated systems. A deployment can exist without any systems attached so I think this is unnecessary. I think it is fine to make the client responsible to also delete the Deployment with a separate request if desired.

For example, if a System was associated to a Deployment by mistake, one may want to remove it, then re-associate the proper system and automatically deleting the Deployment in-between may be an annoyance.

Reply:
> For example, if a System was associated to a Deployment by mistake, one may want to remove it, then re-associate the proper system and automatically deleting the Deployment in-between may be an annoyance.

Sounds like a very reasonable use-case. I think deleting the deployment cascading does not really improve the usability (in the special case that we want to delete both we save 1 request) and only creates a hassle for use-cases such as this one.  I will adjust the spec accordingly. 

Do we need to "discuss" this during the next Telecon? In the referenced Issue is very clear that we previously decided on the behaviour specified here (deleting the deployment).

Reply:
Thanks. I think it's OK to make the change directly.